src/utils/sentry.ts

```diff
@@ -7,22 +7,27 @@ let sentryInitialized = false;
 
 function getAppDetails() {
   const pathName = window.location.pathname.split('/');
-  let appGroup;
+  let appGroup = 'insights';
   let appName;
-  let betaCheck;
-
+  let environment = 'stable';
   if (pathName[1] === 'beta') {
-    betaCheck = ' Beta';
     appGroup = pathName[2];
-    appName = appGroup === 'landing' ? 'landing' : pathName[3];
```

Comment:
This line was weird, but looks like ya lost that landing condition

Reply:
@justinorringer I believe this moved to the else condition down below: https://github.com/RedHatInsights/insights-chrome/pull/3026/files#diff-7cf585363d10fb2f338e86818827620adf01c167e93e8186a776cddf81b8c380R23-R25

Reply:
This will get removed now but yeah it was just lower